CommonApi/src/main/java/mezz/jei/api/recipe/transfer/IRecipeTransferError.java

```diff
@@ -36,6 +36,16 @@ enum Type {
 
 	Type getType();
 
+	/**
+	 * Return the RGBA color of the additional button highlight for {@link Type#COSMETIC}.
```

Comment:
I think this is not RGBA, or else A would be 00 and it wouldn't be visible.
ARGB probably?